status/README.md

```diff
@@ -34,6 +33,14 @@ Ex:
 set -g @catppuccin_date_time_icon "null"
 ```
 
+> [!IMPORTANT]
+> Make sure to load the catppuccin theme prior to setting your status modules.<br/>
+> Use `run ~/.config/tmux/plugins/catppuccin/tmux/catppuccin.tmux`.<br/>
+> For TPM, instead use `run ~/.config/tmux/plugins/tmux/catppuccin.tmux`.
```

Comment:
Should this be an instruction to run tpm?

Reply:
After the issue was closed I did some testing in my config. I found running just the catppuccin command first and then running the TPM command later worked as expected. That made more sense to me than running TPM twice, but I can put that in the docs if that is what you prefer.